distributionviewer/core/static/js/app/components/containers/chart-line-container.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import * as d3Shape from 'd3-shape';
+import * as d3Selection from 'd3-selection';
+
+import ChartLine from '../views/chart-line';
+
+
+export default class extends React.Component {
+  componentDidMount() {
+    let props = this.props;
+    let line = d3Shape.line()
+                .x(d => {return props.xScale(d.x);})
+                .y(d => {return props.yScale(d.y);});
```

Comment:
We can omit the curly braces and the `return` statement since we're doing a one-line arrow function.